shell/browser/native_browser_view_mac.mm

```diff
@@ -213,17 +221,22 @@ - (BOOL)mouseDownCanMoveWindow {
 }
 
 void NativeBrowserViewMac::SetBackgroundColor(SkColor color) {
-  auto* view =
-      GetInspectableWebContentsView()->GetNativeView().GetNativeNSView();
+  auto* contents = GetInspectableWebContentsView();
+  if (!contents)
+    return;
+  auto* view = contents->GetNativeView().GetNativeNSView();
   view.wantsLayer = YES;
   view.layer.backgroundColor = skia::CGColorCreateFromSkColor(color);
 }
 
 void NativeBrowserViewMac::UpdateDraggableRegions(
     const std::vector<gfx::Rect>& drag_exclude_rects) {
+  auto* inspectable_contents = GetInspectableWebContentsView();
```

Comment:
this should be `contents` for consistency although i think it might be clearer to use `auto* iwc_view = GetInspectableWebContentsView();` for this and other instances